platforms/access-bridge/src/http.ts

```diff
@@ -0,0 +1,97 @@
+// Allowed backend request methods.
+export const ALLOWED_REQUEST_METHODS = ['GET', 'PUT', 'POST'] as const;
```

Comment:
This looks very similar to JWPAPIService. Is it possible to move that to a common shared project so it can be used in both places and fully leverage the monorepo setup?

Reply:
@dbudzins , reusing the JWPAPIService as it is now, seems like an overkill. It is coupled with StorageService which we don't need in this case, so leaving that refactor for future improvements.